lib/builders/observability-builder.ts

```diff
@@ -0,0 +1,77 @@
+import * as blueprints from '../../lib';
+import * as utils from "../utils";
+import { NestedStack, NestedStackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class ObservabilityBuilder extends blueprints.BlueprintBuilder {
+
+    public addNativeObservabilityBuilderAddOns(): ObservabilityBuilder {
```

Comment:
Why not add container insights right here? E.g. if the method optionally passes CI config, we can add it right away.

Reply:
removed